tests/query_engines/test_mssql_lib.py

```diff
@@ -2,12 +2,12 @@
 
 import pytest
 import sqlalchemy
+from lib.util import mark_xfail_in_playback_mode
 
 from cohortextractor.query_engines.mssql_lib import (
     ReconnectableConnection,
     fetch_results_in_batches,
 )
-from tests.lib.util import mark_xfail_in_playback_mode
```

Comment:
Needed this import change to run the tests locally; I'm not entirely sure why it works in CI